README.md

```diff
@@ -1112,6 +1112,9 @@ IEEE/RSJ Int. Conf. Intelligent Robots and Systems (IROS), 2023. [PDF](https://a
 *E<sup>2</sup>NeRF: Event Enhanced Neural Radiance Fields from Blurry Images*.
 - [Qi et al., ACM MM 2024](#Qi24acmmm),  
 *Deblurring Neural Radiance Fields with Event-driven Bundle Adjustment*.
+- <a name="li2024benerf"></a>Li, W., Wan, P., Wang, P., Li, J., Zhou, Y., Liu, P.,  
```

Comment:
Could it be Li24eccv if there is no conflict with this key?

Reply:
Sure.